src/view.js

```diff
@@ -129,7 +129,7 @@ Marionette.View = Marionette.AbstractView.extend({
   addRegion: function(name, definition) {
     var regions = {};
     regions[name] = definition;
-    return this._buildRegions(regions)[name];
+    return this.addRegions(regions)[name];
```

Comment:
And FWIW, with @jasonLaster's comment, line 130 and 131 are no longer needed as it'll happen on 137.